boards/arm/96b_carbon_nrf51/96b_carbon_nrf51.dts

```diff
@@ -9,8 +9,7 @@
 
 / {
 	model = "Seeed Studio Carbon nRF51 96board";
-	compatible = "seeed,carbon_nrf51", "nordic,nrf51822-qfac",
-		     "nordic,nrf51822";
+	compatible = "seeed,carbon_nrf51";
```

Comment:
NAK to this without more motivation.

Compatibles are supposed to be listed from most specific to least specific.  Applications and drivers may inspect this to adjust their behavior within a generic implementation to accommodate the needs of a specific one; or to identify that a specific is also compatible with a generic.  In something I was doing today I would have benefitted greatly from being able to tell I was building on an nRF51 rather than an nrRF52: similar compatibles might have let me do that.

Reply:
Those compatibles are still available on the SoC node.  Having the same compatible string for 2 different devices isn't really correct.